woocommerce/wc-admin
# Home Screen - Changed activity panel button styles

This PR is a follow-up of the [PR 5455](https://github.com/woocommerce/woocommerce-admin/pull/5455), so a rebase will be necessary.

This PR changes the styles of the panel button "Manage all orders".

### Screenshots
![screenshot-www figma com-2020 10 28-16_59_03](https://user-images.githubusercontent.com/1314156/97490006-0b690c80-193f-11eb-9560-f1fd145ca33e.png)

### Detailed test instructions:

- Go to the `Home` screen.
- Verify the button `Manage all orders` inside the `Orders` panel is visible.
- Press `Manage all orders` and verify it's working as expected.



### Changelog Note:
Dev: Home Screen - Changed activity panel button styles
